listenbrainz_spark/recommendations/tests/test_models.py

```diff
@@ -0,0 +1,69 @@
+import uuid
+import unittest
+
+import listenbrainz_spark
+from listenbrainz_spark.tests import SparkTestCase
+from listenbrainz_spark import utils, config, hdfs_connection
+import listenbrainz_spark.recommendations.train_models as model
+
+from pyspark.sql import Row
+from pyspark.sql.types import *
```

Comment:
Wildcard imports are an antipattern that I'd like to avoid, can we just import what we need?